users/hillszach/index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Zach-personal
+layout: default
+author: Zach Hills
+---
+# Zachary Hills
+
+<img src="{{ site.url }}users/hillszach/images/37193418_888068498052141_2133433882261848064_o.jpg" height="250" width="250">
+
+Graduate Assistant at Oregon State, Corvallis
+
+M.S. Nuclear Engineering, anticipated 2019
+
+<a href="https://www.linkedin.com/in/zachary-hills-56867873/" target="top"> LinkedIn </a>
+
+Email: hillsz[at]oregonstate[dot]edu
+--------------
+
+## Personal Summary
+
+In the Fall, I am entering my second year of Nuclear Engineering graduate education at Oregon State University under advisor Dr. Todd Palmer. Born in Kansas City and raised in Overland Park, KS, I chose to pursue an undergraduate education in science and technology at the University of Kansas in Lawrence, KS. I began college as a policy debater and learning about carbon driven climate change motivated me to pursue clean energy technology.  My education has explored the air pollution problem and many different research driven ideas for a solution. I have come to understand nuclear technology as most promising for future innovation to cheap, clean energy.
+
+***
+
+## Research Interests
+
+Enhancing our software tools to simulate fuel performance will significantly further efforts to improve expert and public opinions of the market for reactors and ultimately help sell Small Modular Reactors. I am interested in developing the code for particle simulations and multi-physics solvers. 
+
+Currently, I am working in the Reactor Physics group under the advisement of Dr. Mark DeHart at Idaho National Lab. We are developing a I/O linkage program that synchronizes Monte Carlo transport and BISON fuel performance simulations. 
+
+***
+
+## Research Experience
+
```

Comment:
Maybe consider flipping the order of these? I guess it is really personal preference, but it may be more relevant to show your most recent things first. Again, any papers/presentations/reports/etc would be nice links here.

Reply:
I agree with Tony's comment.